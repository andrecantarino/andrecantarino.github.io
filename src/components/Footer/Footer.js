import React from 'react'
import styled from 'styled-components'

const StyledFooter = styled.footer`
  border: 1px solid red;
  display: flex;
  flex-direction: row;
  width: 100%;
  height: 7vh;
`

export const Footer = () => {
  return (
    <StyledFooter>
      <p>Footer</p>
    </StyledFooter>
  )
}
