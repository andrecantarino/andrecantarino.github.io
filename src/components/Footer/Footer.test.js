import React from 'react'
import { render } from '@testing-library/react'
import Footer from './Footer'

describe('Footer | snapshot', () => {
  it('should take a snapshot', () => {
    const { asFragment } = render(<Footer />)

    expect(asFragment(<Footer />)).toMatchSnapshot()
  })
});