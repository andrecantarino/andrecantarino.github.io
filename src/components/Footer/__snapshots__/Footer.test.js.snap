// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer | snapshot should take a snapshot 1`] = `
<DocumentFragment>
  <footer
    class="sc-bczRLJ fPvdEA"
  />
</DocumentFragment>
`;
